freshmaker/errata.py

```diff
@@ -495,6 +495,27 @@ def get_cve_affected_rpm_nvrs(self, errata_id):
 
         return list(nvrs)
 
+    def get_cve_affected_build_nvrs(self, errata_id):
+        """ Get module build nvrs which are affected by the CVEs in errata
+
+        :param errata_id: Errata advisory ID, e.g. 25713.
+        :type errata_id: str or int
+        :return: a list of strings each of them is a module build nvr
+        :rtype: list
+        """
+        data = self._errata_rest_get(f"/erratum/{errata_id}/builds_by_cve")
+        nvrs = set()
+        for data_by_product in data.values():
+            for product_data in data_by_product.values():
+                for build in product_data.get("builds", []):
+                    for build_info in build.values():
+                        if not build_info.get("is_module"):
```

Comment:
Not sure why my comment got lost in previous review comments, re-adding.

This API actually returns only module builds, it's misleading, you should change the API name to indicate that, or I'd suggest something like this:
```
def get_cve_affected_build_nvrs(self, errata_id, is_module=False):
    ...
    if is_module and not build_info.get("is_module"):
        continue
    ...
```

Reply:
Wouldn't the condition be `is_module == build_info.get("is_module", False)` instead? Otherwise it's better to name the argument `only_modules` since `only_modules is False` would mean "both non-modules and module".

Reply:
yes, `only_modules` is a better arg name than `is_module`.

Reply:
Actually I was thinking to add module in the method name, I must make some mistake. But changing in this way is better.